### DISCUSSION TITLE
[Feature] swiping sideways when picture detail menu is up

### DISCUSSION THREAD

**Discussion by RenautMestdagh** [immich-app/immich#5853] [Feature Request]
### The feature

It would be nice if you were able to slide sideways when the picture detail menu is up. (Like with google photos)

https://github.com/immich-app/immich/assets/82072978/7702aac7-a7ad-4460-b50a-e8a677393952

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by alextran1502** [comment]
This is something I want to achieve as well, it remains a UI challenge from GPhotos
